### DISCUSSION TITLE
当我在useFetch headers里面添加大写开头的请求头时，并且请求格式为applications/json时，他会自动转换为小写开头

### DISCUSSION THREAD

**Discussion by usertangxin** [nuxt/nuxt#16963] [Questions]
当我在useFetch headers里面添加大写开头的请求头时，并且请求格式为applications/json时，他会自动转换为小写开头，比如Authorization会被自动转换为authorization，我当前nuxt版本为3.0.0-rc.7

**Comment by caohenghu** [comment]
他这是根据http2协议来的，http2的请求头都是小写，所以框架默认都转小写了。因此你们的后端需要做兼容，对请求头都先转小写再处理
